flake.lock

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Does it make sense if we ignore this file? Sorry, my Nix knowledge is quite limited

Reply:
Huh, I might have misinterpreted a line in [this blog post](https://christine.website/blog/nix-flakes-2-2022-02-27) about what needs to be in source control. Testing...

Reply:
Yup `nix develop` (main thing I'm using it for right now, not actually building with it _yet_) isn't mad at all about gitignoring the lock file. It's removed in 127a13a / a872d05